docs/topics/testing/tools.txt

```diff
@@ -34,12 +34,17 @@ short:
 * Use Django's test client to establish that the correct template is being
   rendered and that the template is passed the correct context data.
 
+* The :class:`~django.test.RequestFactory` is a trimmed down version of the
+  test client that bypasses the routing and middleware layers so you can test
+  your view as you would test any other function in python with known inputs, 
+  testing for specific outputs.
```

Comment:
I think we could be a little more spare here. Maybe: 

```suggestion
* Use :class:`~django.test.RequestFactory` to test view functions directly, 
  bypassing the routing and middleware layers.  
```

Clicking the link you get the explanatory example in the [Advanced testing tools page](https://docs.djangoproject.com/en/4.1/topics/testing/advanced/#the-request-factory).